src/openrct2-ui/windows/Window.h

```diff
@@ -100,7 +100,8 @@ rct_window* window_install_track_open(const utf8* path);
 void window_guest_list_init_vars();
 void window_guest_list_refresh_list();
 rct_window* window_guest_list_open();
-rct_window* window_guest_list_open_with_filter(int32_t type, int32_t index);
+enum class GuestListFilterType : int32_t;
```

Comment:
Sounds a bit weird to have this forward declaration down here, can you please move it up to just before `ScatterToolDensity` forward declaration?